src/a11y-dialog.ts

```diff
@@ -48,7 +50,8 @@ export default class A11yDialog {
     if (destroyEvent.defaultPrevented) return this
 
     // Hide the dialog to avoid destroying an open instance
-    this.hide()
+    if (closeWatcher) closeWatcher.requestClose()
```

Comment:
Second thing I find confusing is the difference between `.close(..)` and `.requestClose(..)`. Honestly not sure I get it.

Reply:
From what I understand, the difference is that `requestClose()` allows the close event to be cancelled (using `preventDefault`) whereas `close()` will force close it.

For the main purpose of `CloseWatcher` (user-initiated close requests), you might be able to get away without using either of these methods.